cw-orch-daemon/src/tx_builder.rs

```diff
@@ -106,7 +106,7 @@ impl TxBuilder {
                 sim_gas_used as f64 * GAS_BUFFER
             };
             let fee_amount = gas_expected
-                * (wallet.daemon_state.chain_data.fees.fee_tokens[0].fixed_min_gas_price + 0.00001);
+                * (wallet.daemon_state.chain_data.fees.fee_tokens[0].average_gas_price + 0.00001);
```

Comment:
The problem with this is that the const configs that we have set the min_gas_price with the price that we set. 

We could start using chain_registry by default?